src/Bridges/Constraint/semi_to_binary.jl

```diff
@@ -0,0 +1,194 @@
+const SemiSets{T} = Union{MOI.Semicontinuous{T}, MOI.Semiinteger{T}}
+
+"""
+    SemiToBinaryBridge{T, S <: MOI.AbstractScalarSet}
+
+The `SemiToBinaryBridge` replaces an Semicontinuous constraint:
+``x \\in Semicontinuous(LowerBound, UpperBound)``
```

Comment:
Multicharacter strings don't print well in LaTeX, use `\mathsf{Semicontinuous}(l, u}` instead